meetings/2025-03-05.md

```diff
@@ -0,0 +1,121 @@
+* JZ: Thursday March 6, 15-17:00 UTC https://meet.jit.si/solid_practioners will focus on Solid and Transportation with talks from Pietr Colpaert and four others working on decentralized transportation.  All invited.
+
+### Connecting a Wallet with a Pod
+* MdJ: Inrupt, KIT and SURF are all experimenting with the connection between pods and wallets. Let's compare notes! https://docs.inrupt.com/data-wallet/patterns-and-flows/
+* MdJ: We are doing it with the Dutch Univeristy. We try to share VCs from my wallet. Instead of it being self contained it refers to a document on the pod. The question is how the signature works then. Does the document contain signature. How does it work with the Inrupt wallet. Can you share a document in the same way as you would share a VC?
+* HZ: Yes you can. It is an API to make the interaction easier. There is an inbox, kind of like a notifications. You can share document, in the mobile app there is a QR code if someone wants to use it. The request is being made to get the document.
+* MdJ: Is anything signed?
+* HZ: ...
+* MdJ: Are you aware of any extension of OIDC4VP that would allow it.
+* HZ: No
+* HZ: It is not that when Inrupt donated it to OWF and wanted it to be the story. It wanted cross pollination and the API to be inproved. We don't care about the wallet but about the storage. It is open source code licensed under Apache.
+* eP: Yesterday I joined the credentials CG, where someone from Octa and Sam from Google were presenting the digital credentials API, and discussed the differences with FedCM. There are multiple protocols, like OpenID4VP. It uses FIDO etc. and QR codes. What is the overlap between that and the wallets you are discussing now? OM th
+* MdJ: I think it is pretty related. AFAIK Credentials CG is about interfacing wallets with the web. You could have a boarding pass or driver's license on your phone, and CG describes how it could be shared in the browser to the website. You share it either with the QR code or with NFC to some device, similar to payments. They also add how to present it via JS API. What I'm doing is about wallets, but not necessarly adding interaces and web page but between them and storage servers.
+* HZ: The wallet API, the backend is documented and open. Did you take a look at it and would it be usefull in your project? It could be useful to have multiple implementations. There is also ane from South American company which is doing attachments on VCs.
+* eP: In the case you're describing, connecting to storage, how is the user involved?
+* MdJ: The user would not see much of a difference between sharing a credential on their device vs sharing one in their storage. You could have them in both location. It could be transparent to the user. The Verifiers would not receive the whole thing in the QR code or NFC interaction. They could receive a read capability for example. 
+* eP: so where the wallet is makes the main difference, device vs pod
```

Comment:
```suggestion
* MdJ: The user would not see much of a difference between sharing a credential on their device vs sharing one in their storage. You could have them in both location. It could be transparent to the user. The Verifiers would not receive the whole thing in the QR code or NFC interaction. They could receive a read capability, for example. 
* eP: So where the wallet is makes the main difference, device vs pod.
```